yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
you have yarn.lock and src/yarn.lock when you should have only src/frontend/yarn.lock or am I missing something?

Reply:
Yes, indeed. Run any yarn command from top level generate this empty lock. I delete it.